supply/lib/supply/options.rb

```diff
@@ -111,7 +111,16 @@ def self.available_options
                                      optional: true,
                                      description: "Whether to skip uploading SCREENSHOTS",
                                      is_string: false,
-                                     default_value: false)
+                                     default_value: false),
+        FastlaneCore::ConfigItem.new(key: :track_promote_to,
+                                     env_name: "SUPPLY_TRACK_PROMOTE_TO",
+                                     optional: true,
+                                     description: "The Track to promote to: production, beta, alpha or rollout",
+                                     verify_block: proc do |value|
+                                       available = %w(production beta alpha rollout)
```

Comment:
Can you extract this as a variable this array of valid tracks is also used [here](https://github.com/sschendel/fastlane/blob/promote_track/supply/lib/supply/options.rb#L19)